src/main/java/com/juniorstart/juniorstart/exception/SaveException.java

```diff
@@ -0,0 +1,27 @@
+package com.juniorstart.juniorstart.exception;
+
+import lombok.Getter;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+/** Class for Save exception.
+ * @author Dawid Wit
+ * @version 1.0
+ * @since 1.0
+ */
+@Getter
+@ResponseStatus(HttpStatus.EXPECTATION_FAILED)
+public class SaveException extends RuntimeException {
```

Comment:
Czy ta klasa jest wgl potrzebna. Wg mnie to powinno iść pod Bad Request/ Conflict
Tylko w tedy musisz przerobić serwis.

Reply:
Też tak uważam

Reply:
Pisałem to wieczorem, z rana stwierdzam, ze na pewno 409 - Conflict.